wsi/drm/drmplane.cpp

```diff
@@ -514,10 +513,17 @@ bool DrmPlane::IsSupportedModifier(uint64_t modifier, uint32_t format) {
     const format_mods& obj = formats_modifiers_.at(i);
     if (obj.format == format) {
       std::vector<uint64_t>::const_iterator it;
+#ifdef OS_YUNOS
```

Comment:
why is this alios specific ?

Reply:
AS YUNOS(alios) toolchain is very old there is compile error here.